miio/integrations/humidifier/deerma/airhumidifier_jsqs.py

```diff
@@ -0,0 +1,235 @@
+        {'did': 'target_humidity', 'siid': 2, 'piid': 6, 'code': 0, 'value': 50},
+        {'did': 'relative_humidity', 'siid': 3, 'piid': 1, 'code': 0, 'value': 40},
+        {'did': 'temperature', 'siid': 3, 'piid': 7, 'code': 0, 'value': 22.7},
+        {'did': 'buzzer', 'siid': 5, 'piid': 1, 'code': 0, 'value': False},
+        {'did': 'led_light', 'siid': 6, 'piid': 1, 'code': 0, 'value': True},
+        {'did': 'water_shortage_fault', 'siid': 7, 'piid': 1, 'code': 0, 'value': False},
+        {'did': 'tank_filed', 'siid': 7, 'piid': 2, 'code': 0, 'value': False},
+        {'did': 'overwet_protect', 'siid': 7, 'piid': 3, 'code': 0, 'value': False}
+    ]
+    """
+
+    def __init__(self, data: Dict[str, Any]) -> None:
+        self.data = data
+
+    # Air Humidifier
+
+    @property
+    def is_on(self) -> bool:
+        """Return True if device is on."""
+        return self.data["power"]
+
+    @property
+    def power(self) -> str:
+        """Return power state."""
+        return "on" if self.is_on else "off"
+
+    @property
+    def error(self) -> int:
+        """Return error state."""
+        return self.data["fault"]
+
+    @property
+    def mode(self) -> OperationMode:
+        """Return current operation mode."""
+
+        try:
+            mode = OperationMode(self.data["mode"])
+        except ValueError as e:
+            _LOGGER.exception("Cannot parse mode: %s", e)
+            return OperationMode.Auto
+
+        return mode
+
+    @property
+    def target_humidity(self) -> Optional[int]:
+        """Return target humidity."""
+        return self.data.get("target_humidity")
+
+    # Environment
+
+    @property
+    def relative_humidity(self) -> Optional[int]:
+        """Return current humidity."""
+        return self.data.get("relative_humidity")
+
+    @property
+    def temperature(self) -> Optional[float]:
+        """Return current temperature, if available."""
+        return self.data.get("temperature")
+
+    # Alarm
+
+    @property
+    def buzzer(self) -> Optional[bool]:
+        """Return True if buzzer is on."""
+        return self.data.get("buzzer")
+
+    # Indicator Light
+
+    @property
+    def led_light(self) -> Optional[bool]:
+        """Return status of the LED."""
+        return self.data.get("led_light")
+
+    # Other
+
+    @property
+    def tank_filed(self) -> Optional[bool]:
+        """Return the tank filed."""
+        return self.data.get("tank_filed")
+
+    @property
+    def water_shortage_fault(self) -> Optional[bool]:
+        """Return water shortage fault."""
+        return self.data.get("water_shortage_fault")
+
+    @property
+    def overwet_protect(self) -> Optional[bool]:
+        """Return True if overwet mode is active."""
+        return self.data.get("overwet_protect")
+
+
+class AirHumidifierJsqs(MiotDevice):
+    """Main class representing the air humidifier which uses MIoT protocol."""
+
+    _supported_models = ["deerma.humidifier.jsqs"]
```

Comment:
```suggestion
    _supported_models = ["deerma.humidifier.jsqs", "deerma.humidifier.jsq5"]
```
It is reportedly working with that device, so worth adding it already here.

Reply:
This works for me and removes the warning I mentioned before.